### DISCUSSION TITLE
values used to determine grain boundary in FIB trace

### DISCUSSION THREAD

**Discussion by Saoussen-ouhiba** [mtex-toolbox/mtex#1058] [Ask Anything]
Hello Rüdiger,

I am sorry for disturbing you again about determining the boundary plane from a 2D EBSD map and a FIB section across a grain boundary. I have exactly a question about the determination of the grain boundary in the FIB trace. 
I am not sure whether the values inc and azi that I am using in the script below are right, knowing that the sample coordinates are as shown in the attached figure. Could you tell me whether this is right please? 

Thank you a lot,

Saoussen

```
inc = alpha; % inclination from the fib section between grain boundary and z axis

azi = beta + pi/2; % trace of fib section (based on x-east / y-south from the EBSD map)

gb_in_fibtrace = vector3d.byPolar(inc,azi)

```
![question](https://user-images.githubusercontent.com/73701277/119673878-05d39680-be3c-11eb-922a-26f02f477eee.JPG)

**Comment by kilir** [comment]
Hi, 
I guess you want to describe the grain boundary normal, right? Hence, I'd say it's alpha + pi/2 or simply take the cross product of the trace defined in the ebsd map and the trace in your fib section  - of course  after making sure both traces are defined in the same coordinate system.
Cheers,
Rüdiger
